modules/tappxBidAdapter.js

```diff
@@ -557,7 +572,7 @@ export function _checkParamDataType(key, value, datatype) {
 
 export function _extractPageUrl(validBidRequests, bidderRequest) {
   // TODO: does the fallback make sense?
-  let url = bidderRequest?.refererInfo?.page || bidderRequest.refererInfo?.topmostLocation;
+  let url = bidderRequest?.refererInfo?.page || bidderRequest?.refererInfo?.topmostLocation;
```

Comment:
pls delete the comment if resolved

Reply:
Hello, thank you for your feedback.
Now the comment is deleted.